ui/js/dfv/src/pods-dfv.js

```diff
@@ -599,6 +599,52 @@ window.PodsDFV = {
 		return true;
 	},
 
+	/**
+	 * Check if Pod being edited is valid.
+	 *
+	 * Returns array of validation Messages
+	 *
+	 * @param {string} pod         Pod slug/name.
+	 * @param {int}    itemId      Object ID.
+	 * @param {int}    formCounter Form index.
+	 *
+	 * @return {string[]} Array of validation messages
+	 */
+	checkValidation( pod, itemId, formCounter ) {
```

Comment:
@sc0ttkclark I don't know how to test this last part, but I think this should work. How do I know which arguments to call? Is there a better way to test?

https://github.com/pods-framework/pods/pull/7064/files#r1173968535